indico/modules/events/abstracts/forms.py

```diff
@@ -61,3 +66,27 @@ class AbstractSubmissionSettingsForm(IndicoForm):
     authorized_submitters = PrincipalListField(_("Authorized submitters"),
                                                description=_("These users may always submit abstracts, even outside "
                                                              "the regular submission period."))
+
+
+class AbstractForm(IndicoForm):
+    title = StringField(_("Title"), [DataRequired()])
+    description = IndicoMarkdownField(_('Content'), [DataRequired()], editor=True, mathjax=True)
+    submitted_contrib_type = QuerySelectField(_("Type"), get_label='name', allow_blank=True,
+                                              blank_text=_("No type selected"))
+    person_links = AbstractPersonLinkListField(_("People"), [DataRequired()])
+    submitted_for_tracks = IndicoQuerySelectMultipleCheckboxField(_("Tracks"), get_label=lambda x: x.title,
+                                                                  collection_class=set)
+    submission_comment = TextAreaField(_("Comments"))
+    attachments = HiddenField(_('Attachments'), widget=DropzoneWidget(style='thin'))
```

Comment:
this should only be visible if attachments are enabled (`del` it in `__init__` otherwise)

`abstract_settings.get('allow_attachments')`